libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -3030,6 +3030,10 @@ def add_iomcu_firmware_defaults(f):
 #ifndef AP_INERTIALSENSOR_ENABLED
 #define AP_INERTIALSENSOR_ENABLED 0
 #endif
+
+#ifndef AP_VIDEOTX_ENABLED
+#define AP_VIDEOTX_ENABLED 0
```

Comment:
It was quite deliberate not to do this since its only the front end and makes the code a little cleaner not doing this.

Reply:
> It was quite deliberate not to do this since its only the front end and makes the code a little cleaner not doing this.

We're including code in builds which is simply not appropriate for a lot of people.  How is that a good thing?  We've got many libraries which we exclude entirely - OSD, for example!

Further, the use of VideoTX will make creation of an AP_Periph which can handle RC data problematic.  And the inclusion of the headers has caused me issues on various projects.